### DISCUSSION TITLE
Disk Unavailable

### DISCUSSION THREAD

**Discussion by artslover1** [cockpit-project/cockpit#19935] [Q&A]
So here is my situation. After installing cockpit i was able to run it fine. But when go to create a RAID, it says no disk available yet it can see all six of the HDD's  For the context I have 3X 10TB and 3X 14TB SATA drives attached via PCie card and they are formatted in exFAT format.

**Comment by jelly** [comment]
Thanks for the feedback! Cockpit currently only shows unformatted disks as "available for RAID". As that's normally how you start, create a raid array and then create a filesystem on it. This is something which is UX wise maybe not ideal and we would like to improve in the future.

**Comment by artslover1** [comment]
Thanks for reply,So if I were to start again how am I to do that? Sorry I am a noob.

**Reply by jelly** [reply]
You can format the disks (if there is no data on it) as "type" => "no filesystem" and then Cockpit should pick it up.

**Reply by artslover1** [reply]
Awesome, it's working, any idea how long will it take? It says 45TB raid device

**Comment by artslover1** [comment]
it would be helpful if there was a section in the installation guide that tells how to create raid and this step is included in it will help save lots of headaches for others as well.

**Reply by artslover1** [reply]
Got every thing working. One other question, how do I make the whole drive visible to every program?

**Reply by jelly** [reply]
You mean, how to mount the raid array?

**Reply by artslover1** [reply]
I was finally able to mount it via Ubuntu but it shows up under /home/sunny/(array name) still the programs don't have access to it. Do I have to mount it via terminal at /mnt for it to be visible?

**Reply by artslover1** [reply]
Got it mounted on /mnt/local/and programs can see the folder now. Thanks a million for all the help.
